sprockets-media_query_combiner.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 1.9.2'
 
-  gem.add_runtime_dependency "sprockets", "~>2.1"
```

Comment:
It's easy to bump it later if it does work, it's hard to pull the gem if it doesn't. I'd rather it was only whitelisted for versions it actually works against.